MonoGame.Framework/SDL/SDLGameWindow.cs

```diff
@@ -218,6 +218,8 @@ public override void EndScreenDeviceChange(string screenDeviceName, int clientWi
                 Sdl.Window.SetFullscreen(Handle, (_willBeFullScreen) ? fullscreenFlag : 0);
                 _hardwareSwitch = _game.graphicsDeviceManager.HardwareModeSwitch;
             }
+            // If going to exclusive full-screen mode, force the window to minimize on focus loss
+            Sdl.SetHint("SDL_VIDEO_MINIMIZE_ON_FOCUS_LOSS", _willBeFullScreen && _hardwareSwitch ? "1" : "0");
```

Comment:
It was tested on my multi-monitor setup here on Windows and works as it should.  Previously it did not remove the window at all, leaving that monitor useless.  If it causes issues on multi-monitor setups on Linux, then they can be fixed later because this is a major issue on Windows right now.